src/main/resources/META-INF/native-image/jni-config.json

```diff
@@ -2,9 +2,5 @@
 {
   "name":"java.lang.Boolean",
   "methods":[{"name":"getBoolean","parameterTypes":["java.lang.String"] }]
-},
-{
-  "name":"sun.management.VMManagementImpl",
```

Comment:
were these changes (and the changes in the two following files) automatically generated?